spec/features/login_signup/signup_spec.rb

```diff
@@ -2,12 +2,17 @@
 RSpec.feature 'SignUp', type: :feature do
   background { visit new_user_registration_path }
   scenario 'has necessary fields' do
-    expect(page).to have_field('Full Name')
     expect(page).to have_field('Email')
+  end
+  scenario 'has necessary fields' do
+    expect(page).to have_field('Full Name')
+  end
+  scenario 'has necessary fields' do
     expect(page).to have_field('Password(6 characters minimum)')
+  end
+  scenario 'has necessary fields' do
     expect(page).to have_field('Confirm Password')
   end
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227711193-bc79ff05-14ed-4c2a-9468-8e6b0cfa8b94.png)

**[OPTIONAL]**

- [ ] It would be great if you make your project summary more descriptive to make your project more professional :+1: